modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -36,6 +36,10 @@
         $fileSize = $_FILES["file"]["size"];
         $fileName = $_FILES["file"]["name"];
 
+        // Escape OS DIRECTORY_SEPARATOR from fileName and user
+        $user     = str_replace(DIRECTORY_SEPARATOR, "", $user);
+        $fileName = str_replace(DIRECTORY_SEPARATOR, "", $fileName);
```

Comment:
I think that it might be a good idea to replace with _ instead of the empty string so that we don't lose (as much?) data.. and why is user coming from the POST request instead of User::singleton?